packages/transaction-controller/src/types.ts

```diff
@@ -100,6 +102,38 @@ export interface Transaction {
   estimateGasError?: string;
 }
 
+/**
+ * @type TransactionReceipt
+ *
+ * TransactionReceipt representation
+ * @property effectiveGasPrice - Effective gas price the transaction was charged at
+ * @property gasUsed - Gas used in the transaction
+ * @property l1Fee - Total used gas in hex
+ * @property status - The status of the transaction
+ */
+export interface TransactionReceipt {
+  blockHash?: string;
+  blockNumber?: number;
```

Comment:
Is this definitely a number, I seemed to remember this coming back as a hex string by default?

Reply:
I was taking the docs of ethers & web3 as source of truth.
Web3: https://github.com/ethers-io/ethers.js/blob/main/src.ts/providers/provider.ts#L895 
Ethers: https://docs.ethers.org/v5/api/providers/types/#providers-TransactionReceipt

Reply:
I also recall seeing this as a `string` in the extension, not sure now if it's the `blockNumber` in `TransactionMeta` or in the `txReceipt`.

Reply:
Ok just checked the API result, this is actually a big number but we will treat this as a string

Reply:
Also the links above is not accurate, this version looks more aligned with the API result https://docs.ethers.org/v6/api/providers/#TransactionReceipt